tools/python/util/pytorch_export_helpers.py

```diff
@@ -0,0 +1,125 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import inspect
+import torch
+
+from collections import abc
+
+
+def _parse_inputs_for_onnx_export(all_input_parameters, inputs, kwargs):
+    # extracted from https://github.com/microsoft/onnxruntime/blob/239c6ad3f021ff7cc2e6247eb074bd4208dc11e2/orttraining/orttraining/python/training/ortmodule/_io.py#L433  # noqa
```

Comment:
>extracted [](http://example.com/codeflow?start=6&length=9)

would we be able to reuse a common helper function eventually?

Reply:
I don't expect so. the issue is the ORTModule stuff is tightly interwoven with training specific logic/needs. I spent a lot of time trying to get to a shared implementation but eventually came to the conclusion that it wouldn't work given we only need a small amount of the existing logic and it's extremely hard to separate cleanly. add to that a common function would need to be in some sort of shared python package which doesn't exist and would need to be created/published/maintained.